cdisutils/dbgap.py

```diff
@@ -122,7 +122,7 @@ def get_project(self, program_name, project_code):
 
         """
 
-        self.logger.info('Looking up project {0}-{1}'
+        self.logger.info('Looking up project {}-{}'
```

Comment:
Since changes are being made elsewhere to use f-strings rather than `str.format`, should the same be done here?